From 51189b7905711dc32592dd64f9c10644fc879a39 Mon Sep 17 00:00:00 2001
From: Frank <464672309@qq.com>
Date: Tue, 6 Feb 2018 16:20:16 +0800
Subject: [PATCH 2/4] =?UTF-8?q?=E5=88=A0=E9=99=A4=E6=96=87=E4=BB=B6?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 text | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 text

diff --git a/text b/text
deleted file mode 100644
index f3a3485..0000000
--- a/text
+++ /dev/null
@@ -1 +0,0 @@
-text
\ No newline at end of file
-- 
2.14.2

